src/IntelliHR/Validation/Validators/MinWords.php

```diff
@@ -26,7 +26,9 @@ public function validateMinWords(
         $this->requireParameterCount(1, $parameters, self::$name);
 
         $minWords = $parameters[0];
-        $wordCount = \str_word_count($value);
+
+        $pregValue = preg_replace('/\s+/', ' ', $value);
```

Comment:
Could see this needing to be a Trait so that anything that counts words does it the same way. Is fine for now